external/source/rdll_template/README.md

```diff
@@ -0,0 +1,27 @@
+# Reflective DLL Project Template
+This is a project template for Microsoft's Visual Studio to aid in the creation
+of Reflective DLLs.
+
+## Installation
+To install the template, copy the template file to the Visual Studio templates
+directory. For a default Visual Studio 2019 installation the command would be:
+
+```
+copy "Reflective DLL.zip" "%USERPROFILE%\Documents\Visual Studio 2019\Templates\ProjectTemplates"
+```
+
+After the template has been copied, restart Visual Studio and select "Reflective
+DLL" from the available project templates.
+
+Ensure that the [ReflectiveDLLInjection][1] submodule has been cloned and is up
+to date with: `git init submodule; git submodule update`.
+
+## Template Updates
+To update the template itself:
+
+1. Open the `rdll_template.sln` file in Visual Studio
```

Comment:
Probably would be better to replace this with the full path to the file, in this case `external/source/rdll_template/rdll_template.sln`, so that people know where to find it easier.

Reply:
Update: Realized that this is in the same folder so am a bit 50/50 about this as I assume if your reading this file you can read were the other files are as well. That being said I do tend to prefer full paths for extra clarity. Up to you but thought I'd just point this out as a discussion point.

Reply:
Eh on second thought when doing this myself it wasn't hard to find the file as assuming you have the readme you can easily locate the .sln file. Marking this as resolved due to it not being as big of an issue as I originally made it out to be in my head.